src/styles/App.scss

```diff
@@ -424,6 +412,31 @@ p {
 p.v-code-text {
   margin-bottom: 10px;
 }
+.alert {
+  --bas-alert-padding-x: 12px;
+  --bas-alert-padding-y: 14px;
+  --bs-alert-padding-x: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  padding-left: 35px;
+}
+.alert-danger {
+  --bs-alert-color: #ff0303;
+  --bs-alert-bg: #fcf0f1;
+  --bs-alert-border-color: #ff9090;
+}
+.alert-danger::before {
+  content: "";
+  background: url(/src/assets/icons/alert-danger.svg) left center no-repeat;
+  display: inline-block;
+  vertical-align: middle;
+  width: 16px;
+  height: 16px;
+  margin-right: 5px;
+  position: absolute;
+  left: 12px;
+}
```

Comment:
can we have a separate scss file for alert styles and import it in App.scss; that way App.scss will not be cluttered

Reply:
done